.github/workflows/pr.yaml

```
@@ -75,7 +75,7 @@ jobs:
         run: make gosec
 
       - name: Upload SARIF file
-        uses: github/codeql-action/upload-sarif@e2b3eafc8d227b0241d48be5f425d47c2d750a13 # v3
+        uses: github/codeql-action/upload-sarif@c36620d31ac7c881962c3d9dd939c40ec9434f2b # v3
```

```suggestion
        uses: github/codeql-action/upload-sarif@f779452ac5af1c261dce0346a8f964149f49322b # v3
```

Diff did not match the PR title for some reason